src/testing/debugging.md

```diff
@@ -25,6 +23,9 @@ Flutter applications. Here are some of the available tools:
 * If you are looking for a way to use GDB to remotely debug the
   Flutter engine running within an Android app process,
   check out [`flutter_gdb`][].
+* [DevTools][], a suite of performance and profiling
+  tools that run in a browser.
+
```

Comment:
I would move this before the Flutter Inspector bullet, since the inspector section assumes awareness of DevTools